packages/ses/test/error/issue-2941/test2/README.md

```diff
@@ -0,0 +1,52 @@
+# Repro multiline error
```

Comment:
We could turn this into a regular ava test (by spawning a process and reading stdout)

Reply:
On my second q, oops. Not yet ready for review.

Reply:
I don't understand my explanation of why this is not yet ready for review. I am making it Ready for Review. I'm also Resolving this conversation. But @naugtur , if you do think I need to make such a change in ***this*** PR, please Unresolve. Thanks.